unlock-app/src/components/interface/prime/PrimeContent.tsx

```diff
@@ -0,0 +1,156 @@
+'use client'
+
+import Head from 'next/head'
+import { pageTitle } from '../../../constants'
+import { TwitterTags } from '../../page/TwitterTags'
+import { OpenGraphTags } from '../../page/OpenGraphTags'
+import { AppLayout } from '../../interface/layouts/AppLayout'
+import { Button, Placeholder } from '@unlock-protocol/ui'
+import { useQuery } from '@tanstack/react-query'
+import { config } from '~/config/app'
+import { Web3Service } from '@unlock-protocol/unlock-js'
+import networks from '@unlock-protocol/networks'
+import { useRouter } from 'next/navigation'
+import { useUnlockPrime } from '~/hooks/useUnlockPrime'
+
+export const PrimeContent = () => {
+  const router = useRouter()
+  const { joinPrime, isPrime } = useUnlockPrime()
+
+  const { data: lock } = useQuery({
+    queryKey: ['prime'],
+    queryFn: async () => {
+      const web3Service = new Web3Service(networks)
+      const lock = await web3Service.getLock(
+        config.prime.contract,
+        config.prime.network
+      )
+      return lock
+    },
+  })
+
+  return (
+    <AppLayout authRequired={false} showLinks={false}>
```

Comment:
we should create a new `layout.tsx` in the `prime` directory to replace this.

it would be something like;

```tsx
export default function PrimeLayout({
  children,
}: {
  children: ReactNode
}) {
  return (
    <div className="overflow-hidden bg-ui-secondary-200">
      <TermsOfServiceModal />
      <Container>
        <PrimeHeader />

        <div className="flex flex-col gap-10 min-h-screen">{children}</div>

        <DashboardFooter />
      </Container>
    </div>
  )
}

Reply:
Should we try to re-use

Reply:
definitely!